core/esmf-test-aspect-models/src/main/resources/valid/samm_1_0_0/org.eclipse.esmf.test/1.0.0/AspectWithConstraintWithSeeAttribute.ttl

```diff
@@ -43,13 +43,15 @@
 :TestTraitTwo a samm-c:Trait ;
    samm:name "TestTraitTwo" ;
    samm-c:constraint [
-      a samm-c:LengthConstraint ;
-      samm:name "TestConstraintTwo" ;
+      a samm-c:RegularExpressionConstraint ;
```

Comment:
Why is the constraint type changed?

Reply:
To test different constraint types (the trait one is already a LengthConstraint), different types of constraints should help identify if the second trait is converted properly in the test.